integration/test_non_vector.py

```
@@ -323,6 +323,162 @@ def validate_aggregate_queries(client: Valkey):
+    # 7. RANDOM_SAMPLE with mixed-type fields (numeric and string in same property)
+    # This tests that RANDOM_SAMPLE handles different types correctly
+    result = client.execute_command(
+        "FT.AGGREGATE", "products", "@price:[1 100]",
+        "LOAD", "2", "price", "rating",
+        "APPLY", "1", "AS", "all",
+        "GROUPBY", "1", "@all",
+        "REDUCE", "RANDOM_SAMPLE", "2", "@price", "3", "AS", "price_sample",
+        "REDUCE", "RANDOM_SAMPLE", "2", "@rating", "3", "AS", "rating_sample"
+    )
+    assert result[0] == 1
+    row = dict(zip(result[1][::2], result[1][1::2]))
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)